regex_1/init.sh

```diff
@@ -0,0 +1 @@
+echo {A..B}{E..F}{I..K}{0..1}{5..6}{I..J}{5..6}{C..D}{1..2}{5..6}{2..3}{4..5}{8..9}{3..4}{6..7}{2..3} > match.txt
```

Comment:
This produce everything will be in a single line. It would be better if it is multiple lines.